keps/sig-instrumentation/1209-metrics-stability/README.md

```diff
@@ -130,15 +130,11 @@ See:
 
 #### Beta -> GA Graduation
 
-- Select stable metrics from control plane components
+- Promote (some) metrics to STABLE status
```

Comment:
I am comfortable with saying "here are some test metrics that are stable, they work, now that this is graduated anyone can confidently pick some" as graduation criteria.